shared/src/commonMain/kotlin/co/touchlab/kampstarter/ktor/DogApiImpl.kt

```diff
@@ -9,7 +10,7 @@ import io.ktor.client.request.HttpRequestBuilder
 import io.ktor.client.request.get
 import io.ktor.http.takeFrom
 
-class DogApiImpl : KtorApi {
+class DogApiImpl(private val log:Kermit) : KtorApi {
     private val client = HttpClient {
```

Comment:
Might be nice to also add a logging feature here and route the ktor logger to kermit